src/Implement/UnitTestHttpListener.cpp

```diff
@@ -0,0 +1,47 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "HttpListener.h"
+
+// Testing HttpListener mock class
+class HttpListenerMock : public HttpListener {
```

Comment:
Usually we put the mock in a separate header.

Reply:
Removed